__tests__/get.test.js

```diff
@@ -209,6 +209,40 @@ describe("GET /api/articles (with query parameters)", () => {
 	});
 });
 
+describe("GET /api/articles (topic filtering)", () => {
+	test("GET:200 articles accept filtering by topic", () => {
+		return request(app)
+			.get("/api/articles?topic=mitch")
+			.expect(200)
+			.then(({ body: { articles } }) => {
+				expect(articles.length).toBe(12);
+				articles.forEach((article) => {
+					expect(article).toHaveProperty("topic", "mitch");
+				});
+			});
+	});
+
+	test("GET:200 responds with empty array if topic does not exist", () => {
+		return request(app)
+			.get("/api/articles?topic=invalid")
+			.expect(200)
+			.then(({ body: { articles } }) => expect(articles).toEqual([]));
+	});
```

Comment:
This is something I would reconsider. 
If the topic exists in the database but has no articles `?topic=paper` , then I would expect the API to respond with an empty array and _200_
However, if the topic does not exist in the database then it might be more useful to respond to the client with an error of `404 - Not Found`  💡